linode_api4/objects/object_storage.py

```diff
@@ -463,8 +465,14 @@ def access(self, cluster, bucket_name, permissions):
         }
 
 
+@deprecated(
```

Comment:
Since we need to deprecate the cluster-view, I assume that we want to deprecate the `ObjectStorageCluster` class. I'm also wondering if we should move the function `buckets_in_cluster` from here to the `obj_storage` group.

Reply:
I think it's a good idea 👍